src/current/_includes/releases/v23.2/v23.2.1.md

```diff
@@ -0,0 +1,161 @@
+- Fixed the value used for the total runtime on SQL statistics. This was using the wrong value previously, causing the [SQL Activity](https://www.cockroachlabs.com/docs/v23.2/ui-overview#sql-activity) page to display values with more than 100%. [cockroachdb/cockroach#117426][#117426]
+- Fixed a bug where trying to set an empty `search_path` [session variable](https://www.cockroachlabs.com/docs/v23.2/session-variables) resulted in an error. [#117557][#117557]
+- It is now possible to assign to the parameter of a [PL/pgSQL](https://www.cockroachlabs.com/docs/v23.2/plpgsql) [routine](https://www.cockroachlabs.com/docs/v23.2/create-procedure). Previously, attempts to do this would result in a "variable not found" error at routine creation time. In addition, variable shadowing is now explicitly disabled, where previously it would cause an internal error. These bugs existed in the v23.2.0 release and the v23.2 pre-release versions. [#117715][#117715]
+- Fixed a bug in the [row-level TTL](https://www.cockroachlabs.com/docs/v23.2/row-level-ttl) [job](https://www.cockroachlabs.com/docs/v23.2/show-schedules) that would cause it to skip expired rows if the [primary key](https://www.cockroachlabs.com/docs/v23.2/primary-key) of the table included columns of the [collated string](https://www.cockroachlabs.com/docs/v23.2/collate) type. This bug was present since the initial release of row-level TTL in v22.2.0. [#117512][#117512]
+- Fixed a bug where concurrent [`GRANT`](https://www.cockroachlabs.com/docs/v23.2/grant) statements can cause deadlocks. [#117713][#117713]
+- CockroachDB can now transparently retry more retryable errors when performing a non-atomic [`COPY`](https://www.cockroachlabs.com/docs/v23.2/copy-from) command. [cockroachdb/cockroach#117895][#117895]
+- Fixed a bug that caused [DML statements](https://www.cockroachlabs.com/docs/v23.2/performance-best-practices-overview#dml-best-practices) to fail while a [hash-sharded index](https://www.cockroachlabs.com/docs/v23.2/hash-sharded-indexes) was being created. The symptom of this bug was an error like `column "crdb_internal_val_shard_16" does not exist`. This bug was present since v23.1.0. [#118215][#118215]
+- Previously, CockroachDB could encounter the error `unable to encode table key: *tree.DTSQuery` when operating on columns with the internal `TSQuery` type in some contexts (e.g. when collecting [table statistics](https://www.cockroachlabs.com/docs/v23.2/cost-based-optimizer#table-statistics) or when performing a [`DISTINCT` operation](https://www.cockroachlabs.com/docs/v23.2/select-clause#eliminate-duplicate-rows)), and this is now fixed. The bug had been present since v23.1 when support for the internal `TSQuery` type was added. [cockroachdb/cockroach#118321][#118321]
```

Comment:
```suggestion
- Previously, CockroachDB could encounter the error `unable to encode table key: *tree.DTSQuery` when operating on columns with the internal `TSQuery` type in some contexts (e.g., when collecting [table statistics](https://www.cockroachlabs.com/docs/v23.2/cost-based-optimizer#table-statistics) or when performing a [`DISTINCT` operation](https://www.cockroachlabs.com/docs/v23.2/select-clause#eliminate-duplicate-rows)). This is now fixed. The bug had been present since v23.1 when support for the internal `TSQuery` type was added. [cockroachdb/cockroach#118321][#118321]
```

Reply:
✅